package array_process.custom_exception;


public class IteratorOutOfBoundsException extends RuntimeException
{

}
